frontend/src/app/rating/rating.page.ts

```diff
@@ -4,38 +4,280 @@
  *
  */
 
-import { Component } from '@angular/core';
+import { Component, OnDestroy, OnInit } from '@angular/core';
 import { Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
 import { NavController } from '@ionic/angular';
+import { AlertController } from '@ionic/angular';
+import { ToastController } from '@ionic/angular';
```

Comment:
With the changes from my PR that have been merged a few minutes ago, the iimports for AlertController and ToastController are blacklisted. Please use the ones I added into common/ion-wrappers/ instead. :)

(At the time you wrote this, my changes weren't merged into master. So you couldn't have known that.)